lib/src/visitor/async_evaluate.dart

```diff
@@ -576,22 +574,21 @@ class _EvaluateVisitor
 
   /// Executes [stylesheet], loaded by [importer], to produce a module.
   ///
-  /// The [configuration] overrides values for `!default` variables defined in
-  /// the module or modules it forwards and/or imports. If it's not passed, the
-  /// current configuration is used instead. Throws a [SassRuntimeException] if
-  /// a configured variable is not declared with `!default`.
+  /// If [configuration] is not passed, the current configuration is used
+  /// instead. Throws a [SassRuntimeException] if a configured variable is not
+  /// declared with `!default`.
   ///
   /// If [namesInErrors] is `true`, this includes the names of modules or
   /// configured variables in errors relating to them. This should only be
   /// `true` if the names won't be obvious from the source span.
   Future<Module> _execute(AsyncImporter importer, Stylesheet stylesheet,
-      {Map<String, _ConfiguredValue> configuration,
-      bool namesInErrors = false}) async {
+      {Configuration configuration, bool namesInErrors = false}) async {
     var url = stylesheet.span.sourceUrl;
 
     var alreadyLoaded = _modules[url];
     if (alreadyLoaded != null) {
-      if ((configuration ?? _configuration).isNotEmpty) {
+      configuration ??= _configuration;
```

Comment:
I don't think you want to make this assignment. We use the fact that this was passed as null later on to determine whether to throw errors about unused configuration variables.